apps/api/src/server/applications/application/application.routes.js

```diff
@@ -78,4 +84,25 @@ router.patch(
 	asyncHandler(updateApplication)
 );
 
+router.get(
+	'/:id?',
```

Comment:
I did this for the test where nothing is entered in ":id" e.g. applications/
I thought it might be worth returning a predefined error

Reply:
Hmmmm I'm not sure what kind of issues this might introduce. The question mark is really an additional piece of information sent in the request, not part of the URL path. I would check in the controller what exactly the request looks like from that point of view, because I get the sense that it might see two question marks one after the other